score-xml/src/main/java/io/cloudslang/content/xml/actions/EditXml.java

```diff
@@ -0,0 +1,191 @@
+     *                        http://xml.org/sax/features/external-general-entities false
+     *                        http://xml.org/sax/features/external-parameter-entities false
+     *                        When the "http://apache.org/xml/features/disallow-doctype-decl" feature is set to "true"
+     *                        the parser will throw a FATAL ERROR if the incoming document contains a DOCTYPE declaration.
+     *                        When the "http://xml.org/sax/features/external-general-entities" feature is set to "false"
+     *                        the parser will not include external general entities.
+     *                        When the "http://xml.org/sax/features/external-parameter-entities" feature is set to "false"
+     *                        the parser will not include external parameter entities or the external DTD subset.
+     *                        If any of the validations fails, the operation will fail with an error message describing the problem.
+     *                        Default value:
+     *                        http://apache.org/xml/features/disallow-doctype-decl true
+     *                        http://xml.org/sax/features/external-general-entities false
+     *                        http://xml.org/sax/features/external-parameter-entities false
+     * @return map of results containing success or failure text, a result message, and the value selected
+     */
+    @Action(name = "Edit XML",
+            outputs = {
+                    @Output(Constants.Outputs.RETURN_RESULT),
+                    @Output(Constants.Outputs.RETURN_CODE),
+                    @Output(Constants.Outputs.EXCEPTION)},
+            responses = {
+                    @Response(text = Constants.ResponseNames.SUCCESS, field = Constants.ReturnCodes.SUCCESS, value = "0"),
+                    @Response(text = Constants.ResponseNames.FAILURE, field = Constants.ReturnCodes.FAILURE, value = "-1")})
+    public Map<String, String> xPathReplaceNode(
+            @Param(value = Constants.Inputs.XML) String xml,
+            @Param(value = Constants.Inputs.FILE_PATH) String filePath,
+            @Param(value = Constants.Inputs.ACTION, required = true) String action,
+            @Param(value = Constants.Inputs.XPATH1, required = true) String xpath1,
+            @Param(value = Constants.Inputs.XPATH2) String xpath2,
+            @Param(value = Constants.Inputs.VALUE) String value,
+            @Param(value = Constants.Inputs.TYPE) String type,
+            @Param(value = Constants.Inputs.TYPE_NAME) String name,
+            @Param(value = Constants.Inputs.FEATURES) String parsingFeatures) {
+        Map<String, String> result = new HashMap<>();
+        try {
+            validateInputs(xml, filePath, action.toLowerCase(), xpath1, xpath2, type, name, parsingFeatures);
+            ActionsEnum myAction = getAction(action);
+
+            switch (myAction) {
```

Comment:
Regulary, the styling for switch blocks is: switch (input) {
  case 1:
break
  case 2:
    prepareOneOrTwo();
    break;
  case 3:
    handleOneTwoOrThree();
    break;
  default:
    handleLargeNumber(input);
}

So many braces for each case are not required.